.vscode/tasks.json

```diff
@@ -0,0 +1,54 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "msbuild: debug",
+      "command": "msbuild",
+      "type": "shell",
+      "args": ["/p:Configuration=Debug", "imgix-csharp.sln"],
+      "group": "test",
+      "problemMatcher": "$msCompile",
```

Comment:
The tasks now use appropriate fields to make this all work without needing `.sh` scripts. The tasks are now also named following typical task / commands conventions.

Reply:
Looks good! 👍